packages/commonwealth/client/scripts/helpers/ContractHelpers/ERC20Helper.ts

```diff
@@ -11,7 +11,11 @@ class ERC20Helper extends ContractBase {
     }
     const balance = await this.contract.methods.balanceOf(userAddress).call();
 
-    return Web3.utils.fromWei(balance, 'ether');
+    const decimals = await this.contract.methods.decimals().call();
+
+    const adjustedBalance = Number(balance) / 10 ** Number(decimals);
```

Comment:
Parsing of onchain integers should always be done via `BigInt`, not `Number`. We've seen multiple crash bugs from integer overflow because of using `Number` in contract helpers and other places.

Reply:
For ERC20 display purposes we need to use number because users have fractional balances(ie bigInt only supports integers). The potential precision issues should be ok for display purposes